product_abc_classification/readme/USAGE.rst

```diff
@@ -0,0 +1,10 @@
+To use this module, you need to:
+
+#. Go to Sales or Inventory menu, then to Configuration/Products/XYZ Analysis Profile
+and create a profile, knowing that the sum of all levels in the profile should sum 100.
```

Comment:
Please, add a comment to explain that the user must to create several levels in that profile (maximum 12, I think)